deployments/multus-daemonset-thick.yml

```diff
@@ -112,11 +112,7 @@ data:
     {
         "chrootDir": "/hostroot",
         "confDir": "/host/etc/cni/net.d",
-        "logToStderr": true,
         "logLevel": "verbose",
-        "logFile": "/tmp/multus.log",
-        "binDir": "/opt/cni/bin",
-        "cniDir": "/var/lib/cni/multus",
```

Comment:
I take it these are defaults now so they don't need to be specified? 

The only advantage is/was that you'd know where those locations are

Reply:
Yeah, these are removed because 

- "binDir" and "cniDir" are same as default value so we don't have to add it, and
- "logFile" is mainly for debug and we already have logs from daemon log (i.e. kubectl log <multus server pod>)

If some user want to change any config, then he/she should read configration guide and know which configuration parameter is the one. Otherwise configuration file should be simple. Otherwise, we should have all configuration parameter in the deployment file (with default value), but this approach seems to be redundant (default parameter is in config file and go code). I believe that the approach is same as various projects (such as kubeadm, apache httpd and so on). They do not have all params in the default config file.

Reply:
thanks for explanation, agreed on removals, and agree we don't have to have all the options filled out.